tools/gke-gpu-driver-version/src/main.rs

```diff
@@ -58,25 +63,38 @@ async fn main() -> anyhow::Result<()> {
             .await?;
 
             let list: GpuDriverVersionInfoList = serde_yml::from_str(&proto)?;
-            let mut result = GpuVersionData::default();
-            result.creation_time = gke_to_cos_versions.creation_time;
-            result.cos_version = v;
-            list.gpu_driver_version_info
-                .into_iter()
-                .filter(|v| match &v.gpu_device {
-                    Some(device) => device.gpu_type == args.gpu_type,
-                    None => false,
-                })
-                .flat_map(|v| v.supported_driver_versions)
-                .for_each(|v| {
-                    if v.label == "DEFAULT" {
-                        result.default_driver_version = v.version.clone();
-                    }
-                    if v.label == "LATEST" {
-                        result.latest_driver_version = v.version.clone();
-                    }
-                });
-            println!("{}", serde_json::to_string_pretty(&result)?);
+
+            match &args.command {
+                Commands::Get(cmd) => {
+                    let mut result = GpuVersionData::default();
+                    result.creation_time = gke_to_cos_versions.creation_time;
+                    result.cos_version = v;
+                    list.gpu_driver_version_info
+                        .into_iter()
+                        .filter(|v| match &v.gpu_device {
+                            Some(device) => device.gpu_type == cmd.gpu_type,
+                            None => false,
+                        })
+                        .flat_map(|v| v.supported_driver_versions)
+                        .for_each(|v| {
+                            if v.label == "DEFAULT" {
+                                result.default_driver_version = v.version.clone();
+                            }
+                            if v.label == "LATEST" {
+                                result.latest_driver_version = v.version.clone();
+                            }
+                        });
+                    println!("{}", serde_json::to_string_pretty(&result)?);
+                }
+                Commands::ListGpuTypes(_cmd) => {
+                    let gpu_types: std::collections::HashSet<String> = list
+                        .gpu_driver_version_info
+                        .into_iter()
+                        .filter_map(|v| v.gpu_device.map(|d| d.gpu_type))
+                        .collect();
```

Comment:
I recommend this list be sorted alphabetically:

```
[
  "NVIDIA_TESLA_A100",
  "NO_GPU",
  "NVIDIA_RTX_PRO_6000",
  "NVIDIA_TESLA_V100",
  "NVIDIA_H200",
  "NVIDIA_A100_80GB",
  "NVIDIA_B200",
  "NVIDIA_H100_80GB",
  "NVIDIA_TESLA_P100",
  "NVIDIA_L4",
  "NVIDIA_TESLA_P4",
  "NVIDIA_GB200",
  "NVIDIA_TESLA_T4",
  "OTHERS"
]
```

This output is a bit haphazard